plugin/trino-druid/src/test/resources/broker-jvm.config

```diff
@@ -3,7 +3,6 @@
 -Xmx256m
 -XX:MaxDirectMemorySize=200m
 -XX:+ExitOnOutOfMemoryError
--XX:+UseG1GC
```

Comment:
Which JDK version does the Druid container run with?

Reply:
1.8 😶‍🌫️